otterdog/eclipse-orbit.jsonnet

```diff
@@ -51,5 +51,14 @@ orgs.newOrg('eclipse-orbit') {
       secret_scanning_push_protection: "disabled",
       web_commit_signoff_required: false,
     },
+    orgs.newRepo('orbit-simrel') {
+      allow_merge_commit: false,
+      allow_update_branch: false,
+      description: "Third-party libraries bundled using Maven target locations.",
+      has_wiki: false,
+      secret_scanning: "disabled",
```

Comment:
I would suggest to enable secret scanning and secret scanning push protection, especially for new repos, if you remove these lines, the values from the default config are used, which enable it.

The same applies for the already existing repos. The config was created with the status quo, but we strongly suggest to enable these settings.

Reply:
Good suggestion. 👍   Done.